scripts/kdb.fish

```diff
@@ -241,8 +241,8 @@ function __fish_kdb_print_resolver_plugins -d 'Print a list of available resolve
 end
 
 function __fish_kdb_print_storage_plugins -d 'Print a list of available storage plugins'
-    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json ni passwd regexstore
-    set -l formats $formats simpleini simplespeclang tcl xmltool uname
+    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json mini ni passwd
+    set -l formats $formats regexstore simpleini simplespeclang tcl xmltool uname
```

Comment:
Sorry for giving a bad tip, I didn't know that performance was an issue.
With 478c7c3c71de076ff2b5032de454c4487dbd4c1d you can use `kdb list storage` which does what you want. On my computer it takes 0,04s (measured with time(1)).